packages/one-app-bundler/__tests__/webpack/loaders/__snapshots__/externals-loader.spec.js.snap

```diff
@@ -1,10 +1,36 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports['externals-loader should ignore the content and get the dependency from the root module 1'] = `
+exports[`externals-loader does not use fallback for server 1`] = `
 "try {
-  module.exports = global.getTenantRootModule().appConfig.providedExternals['my-dependency'].module;
+  const rootModuleExternal = global.getTenantRootModule && global.getTenantRootModule().appConfig.providedExternals['lodash'];
+  if (rootModuleExternal && require('holocron').validateExternal({
+    providedVersion: rootModuleExternal.version,
+    requestedRange: '^1.0.0'
+  })) {
+    module.exports = rootModuleExternal.module;
+  } else {
+    This is CJS code
+  }
 } catch (error) {
-  const errorGettingExternal = new Error('Failed to get external my-dependency from root module');
+  const errorGettingExternal = new Error('Failed to get external lodash from root module on the server', error.message);
+  errorGettingExternal.shouldBlockModuleReload = false;
+  throw errorGettingExternal;
+}
+"
+`;
+
+exports[`externals-loader should ignore the content and get the dependency from the root module 1`] = `
```

Comment:
same thing with this snapshot. I don't see how the snapshot content relates to the intent

Reply:
updated